libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Testing/TestScript.cs

```diff
@@ -128,7 +128,9 @@ public TestAdapter DefaultTestAdapter(ResourceExplorer resourceExplorer, [Caller
                 .UseStorage(storage)
                 .UseBotState(userState, convoState)
                 .Use(new TranscriptLoggerMiddleware(new TraceTranscriptLogger(traceActivity: false)))
-                .Use(new SetTestOptionsMiddleware());
+                .Use(new SetTestOptionsMiddleware())
+                .Use(new SetSkillConversationIdFactoryBaseMiddleware())
```

Comment:
This can be in the test library, rather than Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.TestScript.  Some tests create a custom TestScript: https://github.com/microsoft/botbuilder-dotnet/blob/main/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams.Tests/TestUtils.cs#L33

Or, maybe Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.TestScript.DefaultTestAdapter could use an `IEnumerable<IMiddleware>middleware` parameter, since this seems like a common testing pattern in multiple test projects.